src/test/java/com/google/cloud/compute/v1/it/ITComputeTest.java

```diff
@@ -51,41 +262,951 @@
+  private static final String NETWORK_ENDPOINT_SUB_NETWORK =
+      String.format("%s/subnetworks/default", REGION_LINK);
+
+  private static final ProjectName PROJECT_NAME = ProjectName.of(DEFAULT_PROJECT);
+  private static final ProjectRegionTargetPoolName REGION_TARGET_POOL_NAME =
+      ProjectRegionTargetPoolName.of(DEFAULT_PROJECT, REGION, TARGET_POOL_NAME);
+  private static final ProjectRegionName PROJECT_REGION_NAME =
+      ProjectRegionName.of(DEFAULT_PROJECT, REGION);
+  private static final ProjectZoneName PROJECT_ZONE_NAME =
+      ProjectZoneName.of(DEFAULT_PROJECT, ZONE);
+  private static final ProjectZoneDiskName PROJECT_ZONE_DISK_NAME =
+      ProjectZoneDiskName.of(DISK_NAME, DEFAULT_PROJECT, ZONE);
+  private static final ProjectZoneDiskResourceName PROJECT_ZONE_DISK_RESOURCE_NAME =
+      ProjectZoneDiskResourceName.of(DEFAULT_PROJECT, DISK_NAME, ZONE);
+  private static final ProjectZoneInstanceGroupName INSTANCE_GROUP_NAME =
+      ProjectZoneInstanceGroupName.of(INSTANCE_GROUP, DEFAULT_PROJECT, ZONE);
+  private static final ProjectGlobalBackendBucketName PROJECT_BACKEND_BUCKET_NAME =
+      ProjectGlobalBackendBucketName.of(BACKEND_BUCKET_NAME, DEFAULT_PROJECT);
+  private static final ProjectGlobalFirewallName FIREWALL =
+      ProjectGlobalFirewallName.of(FIREWALL_NAME, DEFAULT_PROJECT);
+  private static final ProjectRegionForwardingRuleName PROJECT_REGION_FORWARDING_RULE_NAME =
+      ProjectRegionForwardingRuleName.of(FORWARDING_RULE_NAME, DEFAULT_PROJECT, REGION);
+  private static final ProjectGlobalHttpHealthCheckName HTTP_HEALTH_CHECK_NAME =
+      ProjectGlobalHttpHealthCheckName.of(HTTP_HEALTH_CHECK_2, DEFAULT_PROJECT);
+  private static final ProjectGlobalHttpsHealthCheckName HTTPS_HEALTH_CHECK_NAME =
+      ProjectGlobalHttpsHealthCheckName.of(HTTPS_HEALTH_CHECK_2, DEFAULT_PROJECT);
+  private static final ProjectGlobalImageName PROJECT_GLOBAL_IMAGE_NAME =
+      ProjectGlobalImageName.of(IMAGE_NAME, DEFAULT_PROJECT);
+  private static final ProjectGlobalImageResourceName IMAGE_RESOURCE_NAME =
+      ProjectGlobalImageResourceName.of(DEFAULT_PROJECT, IMAGE_NAME);
+  private static final ProjectGlobalLicenseName LICENSE_NAME =
+      ProjectGlobalLicenseName.of(LICENSE, DEFAULT_PROJECT);
+  private static final ProjectZoneNetworkEndpointGroupName NETWORK_ENDPOINT_GROUP_NAME =
+      ProjectZoneNetworkEndpointGroupName.of(NETWORK_ENDPOINT_GROUP, DEFAULT_PROJECT, ZONE);
+  private static final Address ADDRESS =
+      Address.newBuilder()
+          .setName(ADDRESS_NAME)
+          .setNetworkTier(NETWORK_TIER)
+          .setRegion(REGION_LINK)
+          .build();
+  private static final TCPHealthCheck TCP_HEALTH_CHECK =
+      TCPHealthCheck.newBuilder().setPort(80).setProxyHeader("NONE").build();
+
+  @Rule public Timeout globalTimeout = Timeout.seconds(1000);
 
   @BeforeClass
-  public static void beforeClass() throws IOException {
+  public static void beforeClass() throws Exception {
+
```

Comment:
Would it make sense to rename beforeClass() to setUp() ? To be consistent with tearDown()

Reply:
yes we can, but you can see [here]( https://github.com/googleapis/java-redis/blob/master/google-cloud-redis/src/test/java/com/google/cloud/redis/it/ITSystemTest.java#L49).

Reply:
May be then rename tearDown() to afterClass()? just to be consistent...

Reply:
done